Packages/io.chainsafe.web3-unity/Runtime/Scripts/Web3Accessor.cs

```diff
@@ -15,9 +15,8 @@ static Web3Accessor Instance
             {
                 if (!instance)
                 {
-                    var go = new GameObject("Web3Accessor");
-                    DontDestroyOnLoad(go);
```

Comment:
why was dont destroy on load removed?

Reply:
This needs to stay alive right?

Reply:
We are in our own example, buuuuuuuuut I would still encourage us to have the web3accessor present for people who are building games 😁

Reply:
understood, reverted